### ISSUE TITLE
[stable/airflow] Airflow NOTES.TXT is not up-to-date

### ISSUE THREAD

**Issue by sarunaskas** [helm/charts#12777]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
Bug

**Which chart**:
stable/airflow

**What happened**:
Template NOTES.TXT hints are not up to date. This is the message received after an install with notable information missing
```Congratulations. You have just deployed Apache Airflow
export POD_NAME=$(kubectl get pods --namespace airflow -l "component=" -o jsonpath="{.items[0].metadata.name}")
echo http://127.0.0.1:
kubectl port-forward --namespace airflow $POD_NAME :

2. Open Airflow in your web browser
```
Runnning the first command does not return results
```
$ export POD_NAME=$(kubectl get pods --namespace airflow -l "component=" -o jsonpath="{.items[0].metadata.name}")
error: error executing jsonpath "{.items[0].metadata.name}": array index out of bounds: index 0, length 0
```

**What you expected to happen**:
The expectation was that the values get filled in according to NOTES.TXT
```
export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "component={{ .Values.airflow.name }}" -o jsonpath="{.items[0].metadata.name}")
   echo http://127.0.0.1:{{ .Values.airflow.externalPortHttp }}
   kubectl port-forward --namespace {{ .Release.Namespace }} $POD_NAME {{ .Values.airflow.externalPortHttp }}:{{ .Values.airflow.internalPortHttp }}
```

**How to reproduce it** (as minimally and precisely as possible):
`helm install --namespace "airflow" --name "airflow" stable/airflow`

**Comment by gsemet** [comment]
Have you time to update it ?

**Comment by MahaGamal** [comment]
This should have been fixed by #12793
Can you try using the newest chart version?

**Comment by sarunaskas** [comment]
@MahaGamal Thanks! The instructions work well with the newest version.

**Comment by MahaGamal** [comment]
Closed #12777
